### PR TITLE
[SDP-125] add fields to serializers for API usage

### PR DESCRIPTION
**Pull request by cjarrett**

### CODE REVIEW

File: app/serializers/api/v1/project_serializer.rb [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
-  attributes :id, :name, :description, :slug, :people_needed, :start_date, :end_date, :city, :state, :zip, :latitude, :longitude, :percent_full
+  attributes :accessible,
+             :address1,
+             :address2,
+             :city,
+             :description,
+             :end_date,
+             :id,
+             :latitude,
+             :leaders,
```
Comment (t2) [inline app/serializers/api/v1/project_serializer.rb]: On the project list there is only 1 leader shown under the project title. Would there be a main leader? Or should we show a list like in the details?

File: db/seeds.rb [hunk 2]
```diff
@@ -65,6 +65,9 @@
     accent_color: Faker::Color.hex_color,
     subdomain: Faker::Cat.name,
     status: "active",
```
Comment (t2) [inline db/seeds.rb]: Can this be updated to "approved" instead of "active"?

File: app/serializers/api/v1/project_serializer.rb [hunk 3]
```diff
@@ -1,5 +1,23 @@
 class Api::V1::ProjectSerializer < ActiveModel::Serializer
-  attributes :id, :name, :description, :slug, :people_needed, :start_date, :end_date, :city, :state, :zip, :latitude, :longitude, :percent_full
+  attributes :accessible,
```
Comment (t2) [inline app/serializers/api/v1/project_serializer.rb]: Where would we determine which age ranges are appropriate per design?

Comment (cjarrett) [reply app/serializers/api/v1/project_serializer.rb]: ah, I see we're using age range for the filters..I'll add.

File: app/serializers/api/v1/project_serializer.rb (hunk 1 above)
Comment (cjarrett) [reply app/serializers/api/v1/project_serializer.rb]: there are co-leaders we can expose through the API. one main leader, multiple co-leaders.

Comment (t2) [reply app/serializers/api/v1/project_serializer.rb]: OK, so the user attached to the project is the main leader? If that's the case, can we get the `belongs_to :user` added to the serializer as well?

**Comment by t2** [comment]
Looks great!

**Pull request merged**